faqrenderer/faq.md

```diff
@@ -15,6 +15,8 @@ The **New Default Environment** is what most users should choose for general Jup
 | C++ (cling) | 0.6 | 0.6 |
 | SageMath | 8.1 | 9.1 |
 
+*Starting with R 4.0, there has been a [change to the behavior of data.frame() and read.table() calls](https://developer.r-project.org/Blog/public/2020/02/16/stringsasfactors/index.html). To maintain compatibility with previous code, you will want to add a `stringsAsFactors=TRUE` parameter to each of your data.frame() and read.table() calls.*
+
```

Comment:
That's an annoying change.

Reply:
Their argument is that different locales might cause different behavior on different machines, but yeah it is a breaking change and kinda annoying.